docs/api-reference/advanced/fuzzy_phrase.mdx

```diff
@@ -49,3 +44,16 @@ SELECT * FROM search_idx.search(
   When set to `true`, **all** tokens of the query have to match in order for a
   document to be considered a match.
 </ParamField>
+
+Setting `match_all_terms` to `true` makes it so that **all** query tokens must
+match in order for a document to be considered a match.
```

Comment:
This suggestion might be a little too rudimentary... but the word `match` might not be a bad entry in your `Concepts` section.

If you really have no idea how our inverted index works, it's not clear what you're matching against (whole document? exact query text in document? one or more tokens?).

We rely on the word `match` quite a bit, and an entry in the `Concepts` section might be a good place to illustrate the index structure.